indico/modules/events/abstracts/templates/display/call_for_abstracts.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {%- trans %}Call for Abstracts{% endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="call-for-abstracts">
+        {% if cfa.is_open %}
+            <table class="dates">
```

Comment:
I find it **very** weird if we need `<table>` elements for layout in 2016. Unless we are trying to make HTML great again.